twilight-gateway/src/stream.rs

```diff
@@ -369,7 +362,7 @@ pub fn start_cluster<F: Fn(ShardId) -> Config>(
     concurrency: u64,
     total: u64,
```

Comment:
This should arguably be a u32 as there's no way someone will use the full u64 range

Reply:
i don't see a need to change it tbh